app/frontend/src/features/communities/CommunityPage/CircularIconButton.tsx

```diff
@@ -1,34 +1,64 @@
-import { IconButton, IconButtonProps, makeStyles } from "@material-ui/core";
+import {
+  IconButton,
+  IconButtonProps,
+  makeStyles,
+  Typography,
+} from "@material-ui/core";
 import React, { ReactNode } from "react";
+import { Link } from "react-router-dom";
 
 const useStyles = makeStyles((theme) => ({
-  root: {
-    borderRadius: "50%",
-    boxShadow: "0 2px 4px",
-    "&:active": {
-      transform: "translateY(1px)",
-      boxShadow: "0 1px 4px",
-    },
+  body: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+  },
+  link: {
+    textDecoration: "none",
+  },
+  button: {
+    marginBottom: theme.spacing(0.5),
   },
 }));
 
 interface CircularIconButtonProps extends IconButtonProps {
   children?: ReactNode;
-  onClick: () => void;
+  label: string;
+  id: string;
+  linkTo?: string;
+  disabled?: boolean;
 }
 
 export default function CircularIconButton({
   children,
-  onClick,
+  id,
+  label,
+  linkTo,
+  disabled = false,
   ...otherProps
 }: CircularIconButtonProps) {
   const classes = useStyles();
-  return (
-    <IconButton
-      {...otherProps}
-      onClick={onClick}
-      className={classes.root}
-      children={children}
-    />
+
+  const body = (
+    <div className={classes.body}>
+      <IconButton
+        {...otherProps}
+        id={id}
+        className={classes.button}
+        children={children}
+        disabled={disabled}
+      />
+      <Typography variant="caption" align="center">
+        <label htmlFor={id}>{label}</label>
+      </Typography>
+    </div>
```

Comment:
```suggestion
      <IconButton
        {...otherProps}
        aria-labelledby={id}
        className={classes.button}
        children={children}
        disabled={disabled}
      />
      <Typography id={id} variant="caption" align="center">
        {label}
      </Typography>
    </div>
```
I don't think this is the right HTML to use, since `label`s are usually used with form inputs. Might be better to go with an `aria-labelledby` with the matching ID attached to the `Typography` component instead as that should work correctly according to the dev tool's accessibility tree tab.